tuneme/templates/base.html

```diff
@@ -53,9 +56,10 @@
           <div class="title">
             <h3>{% trans "Sign out" %}</h3>
           </div>
-          <p>{% trans "Lorem ipsum dolor sit amet, consectetur adipiscing elit." %}</p>
+
           <div class="block turq">
           <div class="signin-form">
+            <p>{% trans "Lorem ipsum dolor sit amet, consectetur adipiscing elit." %}</p>
```

Comment:
Lorem ipsum to be replaced with: "Welcome to TuneMe! Not a member yet? Join us by signing up below. Already signed-up? Then sign in."